iceoryx_examples/callbacks/README.md

```diff
@@ -164,3 +164,7 @@ Afterwards we reset both caches to start fresh again.
         rightCache.reset();
     }
 ```
+
+<center>
+[Check out callbacks on GitHub :fontawesome-brands-github:](https://github.com/eclipse-iceoryx/iceoryx/tree/master/iceoryx_examples/callbacks){ .md-button }
```

Comment:
I like the idea to have a button to jump from the website to the code on github. But I don't like that you can see this raw line when you are on the github readme...

Reply:
I know what you mean. Same goes for the admonitions `!!! warnig`. However, I'd like to stick with the current approach for two reasons:
1) Most readers will probably read it in an IDE, there it's very readable, also other things like formulars with [MathJax](https://squidfunk.github.io/mkdocs-material/reference/mathjax/)
2) The GitHub markdown viewer will not parse these items correctly, with admonitions this is a feature, they strike the eye of the reader and get attention. With the button it does not disrupt the reading flow as it is placed at the very end.
3) Having a single source out weights IMHO all other points